pkg/cmd/controller/test_data/controller_build/completed_generated/cluster_state/pipelineruns.yml

```diff
@@ -10,6 +10,7 @@ items:
       build: "1"
       created-by-prow: "true"
       owner: cb-kubecd
+      pipelineType: meta
```

Comment:
Oh hey, you're right!

...actually, this should probably be prefixed by `jenkins.io/`. Lemme change that now so that we don't have to deal with it in the future.